analyzer/src/main/kotlin/MavenSupport.kt

```diff
@@ -108,6 +109,9 @@ class MavenSupport(localRepositoryManagerConverter: (LocalRepositoryManager) ->
                 validationLevel = ModelBuildingRequest.VALIDATION_LEVEL_MINIMAL
             }
 
+    fun parseLicenses(mavenProject: MavenProject): SortedSet<String> =
```

Comment:
Omit the return type, if possible. That probably also makes the import go away again.

Reply:
Done